cns/types/types.go

```diff
@@ -619,6 +619,17 @@ func init() {
 	types.Add("CnsSnapshotNotFoundFault", reflect.TypeOf((*CnsSnapshotNotFoundFault)(nil)).Elem())
 }
 
+type CnsSnapshotCreatedFault struct {
+	CnsFault
+
+	VolumeId   CnsVolumeId   `xml:"volumeId"`
```

Comment:
I see we have volumeid, datastore and snapshotid for CnsSnapshotCreatedFault

Reply:
Added datastore as well now